webapp/packages/core-blocks/src/useErrorDetails.ts

```diff
@@ -13,11 +13,14 @@ import { DetailsError } from '@cloudbeaver/core-sdk';
 import { errorOf, LoadingError } from '@cloudbeaver/core-utils';
 
 import { ErrorDetailsDialog } from './ErrorDetailsDialog/ErrorDetailsDialog';
+import { useTranslate } from './localization/useTranslate';
+
+type IError = Error | string | null;
 
 interface IErrorDetailsHook {
   name?: string;
   message?: string;
-  error: Error | null;
+  error: IError;
```

Comment:
Not sure that we want this changes, this hook is for displaying details of exception, in that case this hook will do nothing, it's same as pass null as argument

Reply:
We support it in ErrorDetailsDialog
`ErrorDetailsDialog: DialogComponent<Error | string, null>`
also we can pass `DetailsError` as extra prop